<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QTreeView" name="treeView">
          <property name="minimumSize">
           <size>
            <width>326</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="parameters">
          <property name="text">
           <string>parameters</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="Save">
            <property name="text">
             <string>Save Changes</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="0" alignment="Qt::AlignmentFlag::AlignTop">
         <widget class="QPushButton" name="Start">
          <property name="text">
           <string>Start calculation</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QGridLayout" name="gridLayout">
          <item row="5" column="2">
           <widget class="QCheckBox" name="fl_6">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
           <widget class="QLabel" name="com_1">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="fl_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCheckBox" name="fl_5">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="arg_4"/>
          </item>
          <item row="7" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="arg_2"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="com_6">
            <property name="text">
             <string>moment Z</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="fl_1">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
           <widget class="QLabel" name="com_3">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="com_5">
            <property name="text">
             <string>moment Y</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="fl_3">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="arg_1"/>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="arg_pressure"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="arg_6"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="com_4">
            <property name="text">
             <string>moment X</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
           <widget class="QLabel" name="com_7">
            <property name="text">
             <string>pressure</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="arg_5"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="arg_3"/>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="fl_4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
           <widget class="QLabel" name="com_2">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>270</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>select file</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
